web-ng/src/app/components/layer-list/layer-list.component.html

```diff
@@ -20,6 +20,18 @@
   </div>
   <mat-divider></mat-divider>
   <div *ngFor="let entry of layers">
-    <ground-layer-list-item [layer]="entry"> </ground-layer-list-item>
+    <ground-layer-list-item [layer]="entry"></ground-layer-list-item>
+  </div>
+  <div fxLayout="row" class="add-Layer">
+    <button
+      fxFlex="fill"
+      mat-stroked-button
+      color="primary"
+      class="add-Layer-btn"
+      (click)="onAddLayer(':new')"
```

Comment:
Since this function will always be called with the same value, can we can remove this arg?